BlogTemplate/Pages/_LoginPartial.cshtml

```diff
@@ -6,20 +6,11 @@
 @if (SignInManager.IsSignedIn(User))
 {
     <form asp-controller="Account" asp-action="Logout" method="post" id="logoutForm" class="navbar-right">
-        <ul class="nav navbar-nav navbar-right">
-            <li>
-                <a asp-page="/Account/Manage/Index" title="Manage">Hello @UserManager.GetUserName(User)!</a>
-            </li>
-            <li>
+                <a asp-page="/Account/Manage/Index" title="Manage">Owner Settings</a>
                 <button type="submit" class="btn btn-link navbar-btn navbar-link">Log out</button>
-            </li>
-        </ul>
     </form>
 }
 else
 {
-    <ul class="nav navbar-nav navbar-right">
-        <li><a asp-page="/Account/Register">Register</a></li>
-        <li><a asp-page="/Account/Login">Log in</a></li>
-    </ul>
+       <a asp-page="/Account/Login">Log in</a>

```

Comment:
Even when I'm logged in, I'm still able to go to localhost:#####/Account/Login and it lets me login to another account. This link shouldn't be reachable when logged in (and neither should the register link). I'm not sure if we can change this since I think it'd be inside the Identity library, but maybe talk to @jimmylewis about it.